.eslintrc.yaml

```diff
@@ -64,10 +64,12 @@ rules:
     lines-between-class-members: off # be more lenient on member declarations
     max-classes-per-file: off # helper classes are common
     max-len: [warn, 150] # more lenient on max length per line
-    new-cap: off # some class still doesn't follow this yet
-    no-underscore-dangle: off # allow underscores
+    new-cap: error
+    no-underscore-dangle: [warn, {
```

Comment:
I don't know why we have underscored variable names except those for private members? The variables are indeedly invisible if they are not exported.

Reply:
Some temp variables are named like `v3_1`, which I deemed reasonable. but it may be a good idea to keep consistent rules anyways, so I agree to re-enable this rule

Reply:
Got it